components/BottomNavigation/src/MDCBottomNavigationBarController.m

```diff
@@ -100,6 +100,18 @@ - (void)viewDidLoad {
   }
 }
 
+- (void)viewDidAppear:(BOOL)animated {
+  [super viewDidAppear:animated];
+  [self updateNavigationBarInsets];
```

Comment:
why not in viewWillAppear?

Reply:
Looks like this is not necessary.  I've removed it for now and it can be re-added later if we discover a case where it's required.